src/RageUtil.cpp

```diff
@@ -843,6 +843,14 @@ void split( const std::wstring &sSource, const std::wstring &sDelimitor, std::ve
 		do_split( sSource, sDelimitor, asAddIt, bIgnoreEmpty );
 }
 
+std::vector<RString> VectorFromString( const RString& sSource, const char delimiter, const bool bIgnoreEmpty )

```

Comment:
I think let's just call it `split` like the other functions. Honestly ideally they should be named with upper camel case but that's legacy so can be handled it in a separate PR.